services/worker/tests/test_job_manager.py

```diff
@@ -300,6 +299,7 @@ def test_doesnotexist(app_config: AppConfig) -> None:
         job_info=job_info, app_config=app_config, job_runner=job_runner, processing_graph=processing_graph
     )
 
-    assert not job_manager.process()
-    with pytest.raises(DoesNotExist):
```

Comment:
I think test name should change now that it does not throw an exception anymore

Reply:
Oh, I understand it's confusing, but "doesnotexist" referred to the fact that the dataset does not exist, not to the fact that the cache would not be created (thus raising DoesNotExist).
I'll let the name, even if now that we don't reach the Hub, the job manager does now that the dataset does not exist.